conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MsixtyConan(ConanFile):
+    name = "msixty"
+    license = "Proprietary"
+    url = "https://github.com/tradingview/msixty"
+    description = "Making your MSIX experience ten times better!"
+    topics = ("msix", "installer", "bootstrapper")
+
+    settings = "os", "compiler", "build_type", "arch"
+    revision_mode = "scm"
+    exports_sources = [
+        "src/*",
+        "CMakeLists.txt"
+    ]
+    generators = "cmake"

```

Comment:
We should give a try to `CMakeToolchain` + `CMakeDeps`. I will open a separate issue after we merge this PR.